test/runtime/configMatters/comm/remote-taskspawn-stress.chpl

```diff
@@ -0,0 +1,107 @@
+use Time;
+
+config const numSpawns = 1000;
+config const printTimings = false;
+// bogus config const to prevent compiler eliminating tup copy
```

Comment:
Out of curiosity: 'tup' seems like it must be an acronym but I can't turn it into anything.  What does it stand for?

Reply:
Not an acronym, just short for tuple

Reply:
Oh, that's funny.  I knew it was a tuple but didn't snap to that.  Would you mind terribly renaming it to something like `spawnArgs`, so that its name describes what it is rather than how it's implemented?

Reply:
Yeah, good idea.